junit-jupiter-api/junit-jupiter-api.gradle.kts

```diff
@@ -30,3 +30,11 @@ tasks {
 		}
 	}
 }
+
+kotlin {
+	sourceSets {
+		main {
+			languageSettings.optIn("kotlin.contracts.ExperimentalContracts")
```

Comment:
Do consumers of junit-jupiter-api also have to opt-in? If so, what happens of they don't?

Reply:
No, since we opt-in in the `junit-jupiter-api`, consumers don't need to specify any opt-ins in their code